### PR TITLE
Prepare for i18n

### PR DESCRIPTION
**Pull request by tobyhodges**
This adjusts the repository config to connect to CrowdIn for internationalisation.

### CODE REVIEW

File: .github/workflows/sandpaper-main.yaml [hunk 1]
```diff
@@ -32,7 +33,9 @@ jobs:
     steps:
 
       - name: "Checkout Lesson"
-        uses: actions/checkout@v4
+        uses: actions/checkout@v3
```
Comment (tobyhodges) [inline .github/workflows/sandpaper-main.yaml]: Presumably the version should be bumped here? (I copied the contents of https://github.com/swcarpentry-i18n/shell-novice/blob/main/.github/workflows/sandpaper-main.yaml)

File: crowdin.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+preserve_hierarchy: true
+
+files:
+  - source: /episodes/*.Rmd
+    dest: /episodes/%file_name%.md
```
Comment (tobyhodges) [inline crowdin.yml]: @joelnitta this lesson uses R Markdown source files for its episodes. Should the extension here on line 5 be updated to `.Rmd`?

**Comment by tobyhodges** [comment]
@joelnitta: please take a look at my comments above. I am about to disappear for a couple days but I hope that @froggleston will be able to respond and merge in my absence.

File: .github/workflows/sandpaper-main.yaml (hunk 1 above)
Comment (joelnitta) [reply .github/workflows/sandpaper-main.yaml]: Presumably, yes. And the same probably goes for the other i18n repos, which do not seem to have been updated recently (e.g., https://github.com/swcarpentry-i18n/git-novice/blob/92a9370299427603e09189f71e620afe3daa38bc/.github/workflows/sandpaper-main.yaml#L36)

File: crowdin.yml (hunk 2 above)
Comment (joelnitta) [reply crowdin.yml]: Yes, please change that.

**Pull request merged**